graphql_compiler/schema_merging/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql.language.visitor import Visitor, visit
+
+
+class SchemaError(Exception):
+    pass
+
+
+SchemaData = namedtuple('SchemaData', ['query_type', 'scalars', 'directives', 'has_extension'])
```

Comment:
Super micro nit: it's nice to get in the habit of using immutable types whenever mutability is not necessary. For example, that would mean using a tuple instead of a list here. In this case, it doesn't really matter too much one way or the other, so this code is totally fine. I also noticed that you used frozensets elsewhere in the code, so I know you already do this often. I encourage you to continue that, even if both sets of code "work fine", especially when there is no cognitive or visual overhead to one approach over the other -- in this case, it's just a question of `(` vs `[`.